### ISSUE TITLE
Accessibility: Sort items cannot be navigated using up and down arrow keys

### ISSUE THREAD

**Issue by Blackbaud-ScottFreeman** [blackbaud/skyux-lists#47]
### Expected behavior
According to the site we use for accessibility training, you should be able to navigate menu items using up and down arrow keys.  This would also apply to the list sort menu items.
https://teachaccess.github.io/tutorial/#/10

### Actual behavior
List sort items can only be navigated using the tab key.

### Steps to reproduce
Go to the demo page and try navigating through items using the up and down arrow keys.
https://developer.blackbaud.com/skyux/components/sort#demo

### Severity
Low

### Impact
Low

**Comment by Blackbaud-DanMiller** [comment]
Closing issue due to inactivity. If you believe this should still be addressed, please file a new issue and include any additional context and justification. Thank you.
